javascripts/dark-mode.js

```diff
@@ -57,55 +57,66 @@ define([], () => {
     viewModeAnchor.title = 'dark-mode';
   }
 
-  /**
-   * Update the stored value for the current mode
-   * @param {"dark" | "light"} value
-   */
-  function updateValue(value) {
-    window.sessionStorage.setItem('mode', value);
-    lightModeEnabled = value === 'light';
-  }
-
-  function setupDarkModeListener() {
-    var r = document.querySelector(':root');
-    if (!r) {
-      return;
-    }
+  /* eslint block-scoped-var: "off" */
 
+// Wrap your code in a function to handle possible async operations
+async function setupDarkModeListener() {
```

Comment:
`async` is only needed when there's an `await` operation within the function, otherwise it's unnecessary